### ISSUE TITLE
Calvin fine-tuning experiment configuration

### ISSUE THREAD

**Issue by 1650292274** [intuitive-robots/flower_vla_calvin#23]
Hi,

I am trying to replicate the CALVIN abc->d finetune experiment. I want to know if the configuration file contents for different datasets are the same.

These are the two configuration files that I observed on the Calvin dataset.
defaults:
  - callbacks: calvin
  - datamodule: calvin
  - model: flower
  - override hydra/job_logging: colorlog
  - override hydra/hydra_logging: colorlog

root_data_dir: /home/python_project/calvin/dataset/task_ABC_D
lang_folder: lang_clip_resnet50

log_dir: ./logs
slurm: false
seed: 242
device: 'cuda'
batch_size: 1
devices: [0, 1, 3]
act_dim: 7
proprio_dims: 7
obs_dim: 512
goal_dim: 512
obs_seq_len: 1
act_seq_len: 10
multistep: 10 #${act_seq_len}
p_last_state: 0
#gen_img_res: 112
gen_img_res: 112
max_epochs: 35
rollout_lh_skip_epochs: 19
num_workers: 12
benchmark_name: calvin_d # calvin_abcd
use_extracted_rel_actions: true

trainer:
  devices: ${devices}
  precision: bf16-mixed
  max_epochs: ${max_epochs}
  sync_batchnorm: True
  accelerator: gpu
#  strategy: "ddp"
  strategy:
    _target_: pytorch_lightning.strategies.DDPStrategy
    find_unused_parameters: False
    gradient_as_bucket_view: True
    static_graph: True
  limit_train_batches: 1000
  limit_val_batches: 4

#logger:
#  _target_: pytorch_lightning.loggers.WandbLogger
#  save_dir: .
#  name: logger
#  group: mode
#  log_model: false
#  project: ${benchmark_name}
#  entity: bennoq
#  id: ???

logger: null

hydra:
  run:
    dir: ${log_dir}/runs/${now:%Y-%m-%d}/${now:%H-%M-%S}_seed${seed}
  sweep:
    dir: ${log_dir}/runs/${now:%Y-%m-%d}/${now:%H-%M-%S}
    subdir: ${hydra.job.override_dirname}
  job:
    config:
      override_dirname:
        exclude_keys:
          - log_dir
          - datamodule.root_data_dir
          - trainer.gpus
          - datamodule.num_workers
          - trainer.limit_train_batches
          - trainer.limit_val_batches

I don't understand the following several parameters.
root_data_dir: /home/python_project/calvin/dataset/task_ABC_D
lang_folder: lang_clip_resnet50
limit_train_batches: 1000
limit_val_batches: 4
Does this parameter "lang_folder" need to be adjusted according to different datasets? For example, "task_ABC_D" and "task_ABCD_D"can use the same "lang_folder"?

What are the functions of these two parameters "limit_train_batches: 1000" and "limit_val_batches: 4" ? Do we need to adjust them when conducting fine-tuning experiments on the Calvin task_ABC_D ?
Thanks
